CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
```

Comment:
edit comments? I don't think we should allow that.

Reply:
@hasufell One instance where this is useful is the original post of an issue/PR itself, which cannot be deleted. (Unless in some cases if you contact GitHub about it, I think. But it's more convenient to just edit things out. And, for example, especially in cases of doxxing, you'd want to remove things as quickly as possible with your own agency without having to wait and be dependent on GitHub.)

Reply:
@hasufell can you comment on https://github.com/nylira/prism-break/pull/1790#discussion_r147551520? I don't really see the problem especially since, again, GitHub clearly shows the edit history for a post.